### PR TITLE
Email copy

### PR DESCRIPTION
**Pull request by AlannaBurke**
Updating the copy of (I think) all the emails that we are using.

### CODE REVIEW

File: lang/english.php [hunk 1]
```diff
@@ -581,22 +576,24 @@
 </p><p>
 Regards,
 </p><p>
-%SITE_NAME%<br>
+The #DrupalCares Half Million Dollar Homepage Team<br />
 %SITE_URL%<br>';
 $label['forget_pass_email_template']='Dear %FNAME% %LNAME%,
```
Comment (rocketeerbkw) [inline lang/english.php:586]: This template doesn't support HTML. I checked the code and the HTML email should be under the label `forget_pass_email_template_html` which doesn't exist in this language file, so I guess it needs to be added.

**Review by rocketeerbkw — CHANGES_REQUESTED** [review]
The order emails aren't exactly correct IMO. The confirm email goes out BEFORE you pay (aka enter your voucher) and the completed email goes out AFTER you pay/voucher. If we have approvals turned on, then users will not ever get emailed that their pixels are approved/published.

image


For our setup, we could [disable the confirm email](https://github.com/amazeeio/DrupalCaresHalfMillionDollarHomepage/blob/master/config.php#L70), then the user gets just the one after the whole checkout process is finished.

File: lang/english.php (hunk 1 above)
Comment (AlannaBurke) [reply lang/english.php:586]: will fix! some of them had HTML and some didn't, so i wasn't sure. obviously this code is brilliantly commented :D

**Comment by AlannaBurke** [comment]
Oh, frustrating, i was going through the process and trying to see when the email sends, but maybe there was a delay. I think it would make sense to just have one confirmation email.

**Comment by AlannaBurke** [comment]
@rocketeerbkw ok, i think i cleaned up the HTML. I can change up the "complete" email to reflect everything. Would it make sense to also send folks their remaining voucher balance, if any? I wasn't entirely sure how to get that variable. If that's tricky, let's not bother.

**Comment by rocketeerbkw** [comment]
@AlannaBurke ya, let's put it all in the complete email then. It doesn't look easy to add new variables though unfortunately. If you want you can go ahead and [change the config](https://github.com/amazeeio/DrupalCaresHalfMillionDollarHomepage/blob/master/config.php#L70) to disable the confirm email. Also I noticed the label for the HTML forgot password is not correct, it should be `forget_pass_email_template_html` ([source reference](https://github.com/amazeeio/DrupalCaresHalfMillionDollarHomepage/blob/master/users/forgot.php#L119))

**Comment by AlannaBurke** [comment]
@rocketeerbkw i think that should do it!

**Review by rocketeerbkw — APPROVED** [review]

**Pull request merged**